doc/Examples/Earthquake_Visualization.ipynb

```diff
@@ -350,11 +350,6 @@
   }
  ],
  "metadata": {
-  "kernelspec": {
-   "display_name": "Python 2",
-   "language": "python",
-   "name": "python2"
-  },
```

Comment:
Basically yes, lately nb_conda has also been adding specific environment names which I've had to scrub out. It looks like the latest notebook format also adds ``deletable`` and ``editable`` metadata to each cell, so I'll probably have a PR to add those. This is all a giant pain and I'm going to have to write a notebook parsing script that strips all this stuff out.